examples/blog-with-auth/packages/backend/src/plugins/graphql/FieldWithInputPayloadPlugin/global-types.ts

```
@@ -0,0 +1,87 @@
+    export interface PothosKindToGraphQLType {
+      PayloadObject: 'Object';
+    }
+
+    export interface FieldOptionsByKind<
+      Types extends SchemaTypes,
+      ParentShape,
+      Type extends TypeParam<Types>,
+      Nullable extends FieldNullability<Type>,
+      Args extends InputFieldMap,
+      // eslint-disable-next-line @typescript-eslint/no-unused-vars
+      ResolveShape,
```

ESLint

[error] 'ResolveShape' is defined but never used. Allowed unused vars must match /^_/u.

(unused-imports/no-unused-vars)